go/proxyd/config.go

```diff
@@ -59,3 +65,19 @@ type Config struct {
 	RPCMethodMappings map[string]string   `toml:"rpc_method_mappings"`
 	WSMethodWhitelist []string            `toml:"ws_method_whitelist"`
 }
+
+func ReadFromEnvOrConfig(value string) (string, error) {
+	if strings.HasPrefix(value, "$") {
+		envValue := os.Getenv(strings.TrimPrefix(value, "$"))
+		if envValue == "" {
+			return "", fmt.Errorf("config env var %s not found", value)
+		}
+		return envValue, nil
+	}
+
+	if strings.HasPrefix(value, "\\") {
```

Comment:
what the goal of this match?

Reply:
It lets you escape the env vars by prefixing them with a backslash. I have to use two of them here because `\` is Go's escape character as well as TOML's.